resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocator.scala

```diff
@@ -494,10 +525,46 @@ class ExecutorPodsAllocator(
 
 private[spark] object ExecutorPodsAllocator {
 
+  private[spark] val EXIT_MAX_EXECUTOR_FAILURES = 11
```

Comment:
borrowed from YARN module `ApplicationMaster.EXIT_MAX_EXECUTOR_FAILURES`

Reply:
shall we move it to SparkExitCode？